src/MoBi.Core/Commands/UpdateInitialConditionInBuildingBlockCommand.cs

```diff
@@ -32,8 +33,10 @@ public class UpdateInitialConditionInBuildingBlockCommand : BuildingBlockChangeC
          _scaleDivisor = scaleDivisor;
          _negativeValuesAllowed = negativeValuesAllowed;
 
-         var initialCondition = _buildingBlock[_path];
-         if (initialCondition == null) return;
```

Comment:
ok clearly the code above is not equivalent if this returns null when not found.